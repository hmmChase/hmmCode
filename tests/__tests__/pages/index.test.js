import { render, prettyDOM, fireEvent } from '@testing-library/react';
import IndexPage from '../../pages/index';

describe('IndexPage', () => {
  it('matches snapshot', () => {
    const mockProps = {};

    const utils = render(<IndexPage {...mockProps} />);

    const container = utils.container;

    expect(container).toMatchSnapshot();
  });
});
