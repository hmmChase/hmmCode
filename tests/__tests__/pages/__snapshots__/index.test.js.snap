// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`IndexPage matches snapshot 1`] = `
<div>
  <h1>
    Hello World
  </h1>
  <div>
    <button
      class="button button-secondary"
    >
      Click me
    </button>
  </div>
  <div>
    <label>
      <input
        type="checkbox"
      />
      false
    </label>
  </div>
  <div>
    <button>
      Click me
    </button>
    <span>
      You clicked 
      0
       times
    </span>
  </div>
  <div>
    <i
      class="fa-thumbs-up"
    />
  </div>
</div>
`;
